From 5f2e9bea61d1dc7afa9e72bd2d29fa9dc9e68b31 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sun, 4 Jan 2026 23:54:58 +0100
Subject: [PATCH 60/90] arm64: dts: rockchip: rk3399-pinephone-pro: Disable
 interrupt GPIO for WiFi for the Pinephone Pro

This breaks at least dev version of Pinephone Pro. Or use of Pinephone
Pro
with firmware configurations that don't use OOB interrupts.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index 67b28f70b6f9..7a3fa599f541 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -1501,9 +1501,9 @@ &sdio0 {
 	brcmf: wifi@1 {
 		compatible = "brcm,bcm4329-fmac";
 		reg = <1>;
-		interrupt-parent = <&gpio4>;
-		interrupts = <RK_PD0 IRQ_TYPE_LEVEL_HIGH>;
-		interrupt-names = "host-wake";
+		//interrupt-parent = <&gpio4>;
+		//interrupts = <RK_PD0 IRQ_TYPE_LEVEL_HIGH>;
+		//interrupt-names = "host-wake";
 		pinctrl-names = "default";
 		pinctrl-0 = <&wifi_host_wake_l>;
 	};
-- 
2.55.0

